en/simulation/README.md

```diff
@@ -125,6 +125,9 @@ make px4_sitl jmavsim
 > **Note** At some point IO or CPU will limit the speed that is possible on your machine and it will be slowed down "automatically".
 > Powerful desktop machines can usually run the simulation at around 6-10x, for notebooks the achieved rates can be around 3-4x.
 
+<span></span>
```

Comment:
FYI, glitch in gitbook means you need to prefix notes that occur after each other with span as shown.

Reply:
Ok thx.